### ISSUE TITLE
Can't open character sheets

### ISSUE THREAD

**Issue by NahtreX** [LordAshes/TaleSpire-CharacterSheetsPlugin#2]
I'm seeing the following error traying to open the default Jon sheet from an ingame mini:

[Error  : Unity Log] FileNotFoundException: Could not load file or assembly 'System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' or one of its dependencies.
Stack trace:
LordAshes.CharacterSheetsPlugin.Update () (at <bfeb2fa8afd74cb48ff36571da905ba4>:0)

The error is popping uip a lot of times once I try to open the sheet. I believe I'm doing the process correctly (setting the edition to Dnd5e, naming the mini Jon and opening the sheet wjhile selecting the mini and doioing ctrl+o), am I missing something?

**Comment by LordAshes** [comment]
Download the WinForms plugin.

Lord Ashes

**Comment by NahtreX** [comment]
Nice! I think it wasn't on the list of dependencies. It's working amazing now, thanks for the grat mod.

**Comment by LordAshes** [comment]
Yes. It currently doesn't download it automatically because it is not
needed on some systems.

Lord Ashes
